CMakeLists.txt

```diff
@@ -49,29 +51,51 @@ IF(USENVML)
   add_definitions(-DUSENVML)
 ENDIF()
 
-# Because Makefiles are shit to read
+set(CMAKE_CONFIGURATION_TYPES "Release" CACHE STRING "" FORCE)
```

Comment:
I think that part broke the Appveyor build on Windows: The target `ALL_BUILD` seems to include a debug build for which no info can be found.

Reply:
On my system ALL_BUILD works and just creates a `Release` build. Maybe some strange CMake version problem?

Where does the error `read_msr():pread: Input/output error` come from?

Reply:
I've fixed the problem on Windows. We had to add the argument `--config Release` to `cmake`.